packages/lib/gpcircuits/scripts/gen-test-artifacts.sh

```diff
@@ -6,9 +6,9 @@
 set -ex
 
 GPC_NAMES=(
-  "proto-pod-gpc-1o-1e-5md"
-  "proto-pod-gpc-1o-5e-8md"
-  "proto-pod-gpc-3o-10e-8md"
+  "proto-pod-gpc_1o-1e-5md"
+  "proto-pod-gpc_1o-5e-8md"
+  "proto-pod-gpc_3o-10e-8md"
```

Comment:
When they're more stable there should be a table there, yes.  I'm not sure I want anyone actually parsing the filenames or constructing their own, though, since I don't expect the set to be exhaustive.